src/amo/components/CollectionList/index.js

```diff
@@ -55,6 +55,9 @@ export class CollectionListBase extends React.Component<Props> {
           </Card>
           {isLoggedIn &&
             <p className="CollectionList-placeholder">
+              {/* TODO: The collections flow is changing a bit (soon), so this
+              needs to be rechecked once that issue is complete.
+              This placeholder text might be unnessary once that update happens. */}
               {i18n.gettext(
```

Comment:
This should be the "First, create your collection. Then you can add extensions and themes.", text which you originally defined in `Collection/index.js`, but really belongs here.